### PR TITLE
Updated the testcase for edit_families_handler and added a testcase f…

### PR DESCRIPTION
**Pull request by ShifaSZ**
There are three updates:
1. Modify the edit_families_handler api test case by adding more input data to increase coverage.
2. Add a test case for delete_families_handler api.
3. Reformat the whole file and wrap some of the long lines

### CODE REVIEW

File: seqr/views/apis/family_api_tests.py [hunk 1]
```diff
@@ -5,32 +5,62 @@
...
+                {FAMILY_ID_FIELD: '13', 'description': 'Test'}
+            ]
+        }
+        response = self.client.post(url, content_type='application/json',
+                                    data=json.dumps(req_values))
         self.assertEqual(response.status_code, 200)
         response_json = response.json()
 
         self.assertListEqual(response_json.keys(), ['familiesByGuid'])
         self.assertEqual(response_json['familiesByGuid'][FAMILY_GUID]['description'], 'Test')
```
Comment (hanars) [inline seqr/views/apis/family_api_tests.py]: If you are adding additional test cases to the request, please also test that the response contains the updated data. In general, if you modify the input to a test you should also be modifying the test on the response. You should add tests for the additional updated families here

File: seqr/views/apis/family_api_tests.py [hunk 2]
```diff
@@ -5,32 +5,62 @@
...
+                {'familyGuid': FAMILY_GUID}
+            ]
+        }
+        response = self.client.post(url, content_type='application/json',
+                                    data=json.dumps(req_values))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+
+        self.assertListEqual(response_json.keys(), ['individualsByGuid', 'familiesByGuid'])
+        self.assertIsNone(response_json['familiesByGuid'][FAMILY_GUID])
```
Comment (hanars) [inline seqr/views/apis/family_api_tests.py]: whenever you test deleting, you should test that trying to delete the same item again fails

File: seqr/views/apis/family_api_tests.py (hunk 1 above)
Comment (ShifaSZ) [reply seqr/views/apis/family_api_tests.py]: Okay. I will add response checking for the updated data.

File: seqr/views/apis/family_api_tests.py (hunk 2 above)
Comment (ShifaSZ) [reply seqr/views/apis/family_api_tests.py]: The deletion looks successful for the item doesn't exist. So I will not try to delete it again.

Comment (ShifaSZ) [reply seqr/views/apis/family_api_tests.py]: But I will update the test to cover more lines in the code.

File: seqr/views/apis/family_api_tests.py [hunk 3]
```diff
@@ -5,31 +5,69 @@
...
         _check_login(self, url)
 
-        # send request
-        req_values = {'families': [{'familyGuid': FAMILY_GUID, 'description': 'Test'}]}
-        response = self.client.post(url, content_type='application/json', data=json.dumps(req_values))
+        # send request with a "families" attribute
+        req_values = {
+            'families': [
+                {'familyGuid': FAMILY_GUID, 'description': 'Test description 1'},
+                {PREVIOUS_FAMILY_ID_FIELD: '2', 'description': 'Test description 2'},
```
Comment (hanars) [inline seqr/views/apis/family_api_tests.py]: So the previous ID field is designed to be used to change the ID, so the usage here is not actually testing a realistic use-case. You should have the previous ID as well as a new ID, and then you should check in the response that the ID for this individual has been successfully updated

Comment (ShifaSZ) [reply seqr/views/apis/family_api_tests.py]: Okay. Updated.

File: seqr/views/apis/family_api_tests.py [hunk 4]
```diff
@@ -59,48 +98,77 @@ def test_update_family_pedigree_image(self):
...
+        self.assertListEqual(response_json.keys(), ['info', 'errors', 'warnings', 'uploadedFileId'])
+
+        url = reverse(edit_families_handler, args=[PROJECT_GUID])
+
+        response = self.client.post(url, content_type='application/json',
+                data=json.dumps({'uploadedFileId': response_json['uploadedFileId']}))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+
+        self.assertListEqual(response_json.keys(), ['familiesByGuid'])
```
Comment (hanars) [inline seqr/views/apis/family_api_tests.py]: Can you please add some tests that the family data here is correct

Comment (ShifaSZ) [reply seqr/views/apis/family_api_tests.py]: Response data checking is added.

**Pull request merged**